pkg/codegen/go/doc.go

```diff
@@ -90,11 +89,12 @@ func (d DocLanguageHelper) GetDocLinkForFunctionInputOrOutputType(pkg *schema.Pa
 }
 
 // GetLanguageTypeString returns the Go-specific type given a Pulumi schema type.
-func (d DocLanguageHelper) GetLanguageTypeString(pkg *schema.Package, moduleName string, t schema.Type, input bool) string {
-	modPkg, ok := d.packages[moduleName]
+func (d DocLanguageHelper) GetTypeName(pkg *schema.Package, t schema.Type, input bool, relativeToModule string) string {
+	goPkg := moduleToPackage(pkg.Reference(), d.goPkgInfo.ModuleToPackage, relativeToModule)
+	modPkg, ok := d.packages[goPkg]
 	if !ok {
-		glog.Errorf("cannot calculate type string for type %q. could not find a package for module %q", t.String(), moduleName)
-		os.Exit(1)
+		glog.Fatalf("cannot calculate type string for type %q. could not find a package for module %q",
```

Comment:
Was it a bug that we were using `Errorf` before?

Reply:
Ah I see -- is `Fatalf` just error with the exit for free?

Reply:
Yeah, except it tries to dump a stacktrace instead of calling `os.Exit`.